tests/spark/test_spark_model_export.py

```diff
@@ -984,3 +986,60 @@ def test_model_log_with_signature_inference(spark_model_iris, input_example):
     else:
         assert column_names == ["0", "1", "2", "3"]
     assert mlflow_model.signature.outputs == Schema([ColSpec(type=DataType.double)])
+
+
+def test_model_with_vector_input(model_path, spark):
+    from mlflow.types.schema import Array
+
+    mlflow.pyspark.ml.autolog()
+    train_df = spark.createDataFrame(
+        [([3., 4.], 0), ([5., 6.], 1)], schema="features array<double>, label long"
+    ).select(array_to_vector("features").alias("features"), col("label"))
+
+    lor = LogisticRegression(maxIter=2)
+    with mlflow.start_run() as run:
+        lor.fit(train_df)
+
+    model_uri = f"runs:/{run.info.run_id}/model"
+
+    # check vector type is inferred correctly
+    model_info = mlflow.models.get_model_info(model_uri)
+    input_type = model_info.signature.inputs.input_dict()['features'].type
+    assert isinstance(input_type, Array) and input_type.is_sparkml_vector
```

Comment:
```suggestion
    assert isinstance(input_type, Array)
    assert input_type.is_sparkml_vector
```

`assert` should not contain multiple conditions.